### PR TITLE
add tickets page

### PR DESCRIPTION
**Pull request by danjo3**

### CODE REVIEW

File: src/css/tickets.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+ /* announcements bar */
+.announcements-container {
+  display: flex;
+  align-items: center;
+  background-color: rgb(231, 211, 31);
+  padding-top: 5px;
+  padding-bottom: 5px;
+}
+
+.announcements-container > div {
```
Comment (dsbrgg) [inline src/css/tickets.css:13]: Maybe try to give the `announcements-container`  children a class of itself like `.announcements-container-child`

File: src/css/tickets.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    padding: 15px 60px;
+    text-align: left;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 4px 2px;
+	background-color: #bc3434;
+}
+
+.price{
```
Comment (dsbrgg) [inline src/css/tickets.css:42]: Try to be more verbose on classes/ids: `tickets-price-label`  or something like that.

File: src/css/tickets.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  text-align: right;
+  text-decoration: none;
+  font-size: 18px;
+  margin: 4px -50px;
+}
+
+
+ /* hero image and text */
+.hero-image {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("../public/images/img2.jpg");
```
Comment (dsbrgg) [inline src/css/tickets.css:54]: Put `url`  in the next line.

File: src/css/tickets.css [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+}
+
+ /* payment and contact information */
+.payment-contact-container {
+  display: flex;
+  align-items: stretch;
+  background-color: DodgerBlue;
+  width: 100%;
+  padding: 50px 50px 30px 20px;
+
```
Comment (dsbrgg) [inline src/css/tickets.css:81]: Remove empty line.

File: src/css/tickets.css [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+  color: rgb(2, 2, 2);
+  border: 4px black;
+  margin: 30px;
+  font-size: 30px;
+  border: 2px solid rgb(0, 0, 0);
+  width: 80%;
+  height: 80%;
+}
+
+.map-image {
```
Comment (dsbrgg) [inline src/css/tickets.css:115]: Try to remove this class possibly? Seems like `.map-container` should already be enough for the children elements.

File: src/pages/tickets.html [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> <!--social favigon-->
+  <link rel="stylesheet" type="text/css" href="../css/tickets.css">
+</head>
+
+<!--Ticket page body-->
+<body>
+    <!--header section-->
+	
+    <!--navbar--> 
+    <header class="sticky" style="z-index: 9999;">
```
Comment (dsbrgg) [inline src/pages/tickets.html:23]: Try to move the `z-index`  to the `sticky`.

File: src/pages/tickets.html [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+		<div style="flex-grow: 4"></div>
+		<div style="flex-grow: 0.1">Purchase in App and save 10%!</div>
+	</section>
+
+    <!--main-->
+    <main id="main-container">
+        <!--ticket information-->
+        <section class="ticket-container">
+            <div class="ticket-label">
+                <div><h1>Option One</h1></div>
```
Comment (dsbrgg) [inline src/pages/tickets.html:52]: Probably don't need these `div`  tags wrapping `h1`  and `span`  tags.

File: src/pages/tickets.html [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+                    Integer vel feugiat sapien, sed tempor augue.
+                    ras ut sapien eu arcu convallis accumsan.
+                    In eget ante et est sollicitudin venenatis. Duis nec euismod odio, non ultrices nibh..</span></div>
+                <div class="price"><h1>$10</h1></div>
+            </div>
+        </section>
+        
+        <section class="ticket-container">
+            <div class="ticket-label">
+                <div><h1>Option Two</h1></div>
```
Comment (dsbrgg) [inline src/pages/tickets.html:65]: Same as above.

File: src/pages/tickets.html [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+        <section class="ticket-container">
+            <div class="ticket-label">
+                <div><h1>Option One</h1></div>
+                <div><span><b>Information:</b></span></div>
+                <div><span>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed ac facilisis metus. 
+                    Donec faucibus vel lorem vitae cursus. 
+                    Integer vel feugiat sapien, sed tempor augue.
+                    ras ut sapien eu arcu convallis accumsan.
+                    In eget ante et est sollicitudin venenatis. Duis nec euismod odio, non ultrices nibh..</span></div>
+                <div class="price"><h1>$10</h1></div>
```
Comment (dsbrgg) [inline src/pages/tickets.html:59]: You could try to move the `class`  to the `h1`  tag, see if it behaves accordingly and if the display is not the same you could try setting `display: block` on the styling for `price`.

File: src/pages/tickets.html [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+                    Integer vel feugiat sapien, sed tempor augue.
+                    ras ut sapien eu arcu convallis accumsan.
+                    In eget ante et est sollicitudin venenatis. Duis nec euismod odio, non ultrices nibh.</span></div>
+                <div class="price"><h1>$20</h1></div>
+            </div>
+        </section>
+
+        <section class="ticket-container">
+            <div class="ticket-label">
+                <div><h1>Option Three</h1></div>
```
Comment (dsbrgg) [inline src/pages/tickets.html:78]: Same as above.

File: src/pages/tickets.html [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+                    In eget ante et est sollicitudin venenatis. Duis nec euismod odio, non ultrices nibh.</span></div>
+                <div class="price"><h1>$30</h1></div>
+            </div>
+        </section>
+        <!--payment and contact information-->
+        <div class="payment-contact-container">
+            <div style="flex-basis:60%">
+				<!-- animated mascot -->
+				<div class="flexbox-media mascot">
+				<img src="../public/images/mascot_crop.gif" style="flex-grow: 1; float: right; width: 20%; height 20%">
```
Comment (dsbrgg) [inline src/pages/tickets.html:93]: Try to move any `style`  (inline styling) to its own class/id.

File: src/pages/tickets.html [hunk 12]
```diff
@@ -0,0 +1,139 @@
...
+                <div class="logo-container"><img src="../public/images/shape_logo_circ.png" class="logo"></div>
+                <div class="menu-container"><b>Attractions &amp; Facilities</b></div>
+                <div class="menu-container"><b>Ticket info</b></div>
+            </nav>
+        </nav>
+    </header>
+	
+	<!--hero img-->
+	<header class="hero-image">
+		<div class="hero-text">
```
Comment (dsbrgg) [inline src/pages/tickets.html:35]: Maybe try to use  a `span`  here? We also might be better off using a `flex`  container on `hero-image`  and then remove the `display: absolute`  and try to position it only with flex properties.

**Comment by danjo3** [comment]
todo: remove unnecessary padding around map and try to make sections on page closer together vertically.

**Pull request closed without merging**